public/css/blog.css

```diff
@@ -0,0 +1,69 @@
+/* General styles */
+body {
+    font-family: 'Open Sans', sans-serif;
+    margin: 0;
+    padding: 0;
+    background-color: rgb(222, 221, 221);
+  }
+  
+  /* Main Content */
+  main {
+    padding-top: 80px; 
+  }
+  
+  /* Form */
+  form.post-form {
+    background-color: #fffefc;
+    border-radius: 8px;
+    box-shadow: 0 4px 8px rgba(0, 0, 0, 0.05);
+    padding: 30px;
+    margin: 0 auto; 
+    max-width: 600px;
+  }
+  
+  .form-group {
+    margin-bottom: 15px;
+    display: flex; 
+    align-items: center;
+  }
+  
+  form label {
+    font-weight: bold;
+    color: #666;
+    margin-right: 10px; 
+  }
+  
+  form input[type="text"],
+  form textarea {
+    border: 1px solid #ddd;
+    border-radius: 4px;
+    padding: 10px;
+    font-size: 1em;
+    transition: all 0.3s ease;
+    width: 100%; 
+  }
+  
+  form input[type="text"]:focus,
+  form textarea:focus {
+    border-color: #999;
+    box-shadow: 0 0 5px rgba(0, 0, 0, 0.1);
+  }
+  
+  form textarea {
+    height: 200px;
+  }
+ 
+  button[type="submit"] {
+    background-color: #333;
+    color: white;
+    padding: 12px 20px;
+    border: none;
+    border-radius: 4px;
+    cursor: pointer;
+    transition: background-color 0.3s ease;
+  }
+  
+  button[type="submit"]:hover {
+    background-color: #000000;
+  }
+  
```

Comment:
some places used hex code and some places rgb why? Do you know the purpose

Reply:
Hex codes do not natively support transparency, whereas RGB can be extended to RGBA to include transparency.